src/main/java/core/basesyntax/FruitShopApp.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public class FruitShopApp {
+    public static void main(String[] args) {
+        try {
+
+            List<FruitTransaction> transactions = CsvReader
+                    .readTransactions("src/main/resources/input.csv");
+
+            transactions = DataProcessor.processTransactions(transactions);
+
```

Comment:
See main method structure. We should have 5 steps but I see only 3 ones.
Also, there is no DB in your app - it is required.
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/c63310df-2e07-4d2e-92ab-2ae996aa4520)